src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig

```diff
@@ -187,6 +187,14 @@
                                                     <div class="tab">
                                                         <h3 class="tab-title">Attachment #{{ loop.index }}</h3>
                                                         <div class="tab-content">
+                                                            {% set filename = attachment.filename|default() %} {# symfony/mime < 6.1 would resolve to null #}
+
+                                                            {% if filename %}
```

Comment:
without a filename we could still have a downloadable body isnt it?

Reply:
True.. what do you think we should display as the filename/link text?

Reply:
dunno .. what's in store? :D

Reply:
What do email clients show for an attachment without a filename?

Reply:
`<em>(no filename)</em>` will generally work :+1: